kairos-test-utils/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "kairos-test-utils"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+
+[dependencies]
+
+log = "*"
+tokio = { version = "*", features = [ "full", "macros" ] }
+env_logger = "*"
```

Comment:
I'd prefer to uniformly use `tracing` over `log`/`env_logger`.
This can be changed in a followup eventually.

Reply:
replaced log with tracing